internal/http/services/owncloud/ocdav/move.go

```diff
@@ -266,7 +266,11 @@ func (s *svc) handleMove(ctx context.Context, w http.ResponseWriter, r *http.Req
 		}
 		// TODO what if intermediate is a file?
 	}
-
+	// resolve the destination path
+	if dst.Path == "." {
+		dst.Path = utils.MakeRelativePath(dstStatRes.GetInfo().GetName())
+		dst.ResourceId = dstStatRes.GetInfo().ParentId
```

Comment:
```suggestion
		dst.ResourceId = dstStatRes.GetInfo().GetParentId()
```
My IDE gets angry when we don't use getters :(